### ISSUE TITLE
It won't let me tag my abuser?

### ISSUE THREAD

**Issue by Tootfroot** [fabacab/pat-facebook#95]
# Noverse Bug Reporting Template

Title:    

Class:
"System Error"

Date:           27/01/2016
Reported By:  
Email:      

Product:                        Version:
Platform:                       Version:
Browser:                        Version:
URL:        

Is it reproducible: Yes
# Description

The PAT will not show my abuser in search results or allow me to tag him in my own story, wether I use his fb ID, user name, or url. 
## Steps to Produce/Reproduce

Enter his name, url, or his ID number and he will either not appear in results, or the tool will tell me that it "does not exist", when I know it does as we are still facebook friends.
## Expected Results

See above
## Actual Results

As expected
## Workarounds

None I can find
## Attachments

None.
## Other Information

None.

**Comment by fabacab** [comment]
Yeah, PAT Facebook is broken until someone updates it to use Facebook's v2 API (see #94). I no longer maintain this software because of a lack of support and due to active personal attacks by rapists, rape apologists, and (most disappointingly) self-ID'd anti-abuse activists.

See fabacab/pat-okcupid#21 for a discussion about that.

**Comment by Tootfroot** [comment]
Oh, that's a shame :c Thank you for trying though c: The heckling must've
been a huge pain

**Comment by meitar** [comment]
The heckling isn't the thing that killed this project. It was the lack of meaningful, visible, public support.
